src/libraryEditButton.tsx

```diff
@@ -1,21 +1,44 @@
 import React from "react";
-import { Button } from "react-bootstrap";
+import { Button, Dropdown, ButtonGroup } from "react-bootstrap";
 import { Book } from "./ourtypes";
+import styled from "styled-components";
+
+const DangerDropdownItem = styled(Dropdown.Item)`
+  background-color: red;
+  color: white;
+  :hover {
+    color: red;
+  }
+`;
```

Comment:
I would suggest the following styling to maintain the bootstrap look. I copied the color codes from the "danger-outline" variant of button.
```css
  background-color: white;
  color: #dc3545;
  border-radius: 3pt;
  border-color: #dc3545;

  :hover {
    color: white;
    background-color: #dc3545;
  }
```